integr8ly/installation
# Update launcher_catalog_git_ref for 1.3.0 RC2

## Additional Information
JIRA: https://issues.jboss.org/browse/INTLY-1540

P.S: I manually created referenced tag: https://github.com/integr8ly/launcher-booster-catalog/releases/tag/integreatly-1.3.0 . This will be automated in the future.

## Verification Steps
Wait until https://github.com/integr8ly/fuse-flights-aggregator/pull/3 is merged.
Choose a cluster with nexus installed in a namespace with prefix (e.g. openshift-nexus).
Edit launcher config map to use `release-1.3.0-rc2` as value for `launcher.backend.catalog.git.ref` field.
Create and launch fuse-flights-aggregator from your launcher.
The app should build successfully.
